test/integration/filestore/gcs/gcs_test.go

```diff
@@ -195,23 +195,3 @@ func TestList(t *testing.T) {
 		})
 	}
 }
-
-// Check if it panics when connect with closed client.
-func TestClose(t *testing.T) {
```

Comment:
Why do we need to remove this test? Did the library change its behavior? 🤔

Reply:
Yes, it was changed to not cause panic when the client already was closed, but it's not big deal for us.